### PR TITLE
Integrated Mayhem

### PR DESCRIPTION
**Pull request by 4812571**
Integrated Mayhem, an automated fuzz testing solution

### CODE REVIEW

File: .github/workflows/mayhem.yml [hunk 1]
```diff
@@ -0,0 +1,74 @@
+name: Mayhem
```
Comment (xansec) [inline .github/workflows/mayhem.yml:1]: Can you please use the latest workflow example here: https://github.com/mayhemheroes/mayhem-cmake-example/blob/solution/.github/workflows/mayhem.yml

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

**Comment by 4812571** [comment]
Changes made @xansec!

**Comment by 4812571** [comment]
Also, do you know how I could squash these commits? It failed for me because there is a merge among them.

**Comment by 4812571** [comment]
I also had some missing binaries in my cargo manifest. I have added those as well

**Review by xansec — APPROVED** [review]
I added the merge commits. LGTM!

**Pull request merged**